int16[] Q_matrix_pseudo_inverse #only elements for torque, rotor_num * 3 matrix
int16[6] inertia #xx, yy, zz, xy, yz, zx
